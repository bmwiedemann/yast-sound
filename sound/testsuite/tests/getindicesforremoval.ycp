{
    include "testsuite.ycp";
    include "sound/routines.ycp";

    TEST(``(get_indices_for_removal([$["module":"A"], $["module": "B"]], "A", 0)), [], nil);
    TEST(``(get_indices_for_removal([$["module":"A"], $["module":"A"], $["module":"B"]], "A", 1)), 
	[], nil);
    TEST(``(get_indices_for_removal([$["module":"A"], $["module":"A"], $["module":"B"]], "B", 2)),
	[], nil);
    TEST(``(get_indices_for_removal([$["module":"A"], $["module":"A"], $["module":"B"], $["module":"B"]], "B", 3)),
        [], nil);
}
