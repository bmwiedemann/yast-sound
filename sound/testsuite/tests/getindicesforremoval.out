Return	[0]
Return	[0, 1]
Return	[2]
Return	[3]
